src/graph_notebook/magics/graph_magic.py

```diff
@@ -13,11 +13,15 @@
 import datetime
 import os
 import uuid
+import requests
 from enum import Enum
 from json import JSONDecodeError
 from graph_notebook.network.opencypher.OCNetwork import OCNetwork
 
 import ipywidgets as widgets
+import pandas as pd
+import itables.options as opt
```

Comment:
Do these work OK with the classic notebooks or will we still use the current templates there?

Reply:
Yes, this works on both classic notebooks and JupyterLab, and is now the first option for both. Itables does not work offline, so in cases where there is no internet connectivity, the current templates will be used instead.